src/dimensionEntry.js

```diff
@@ -3,25 +3,31 @@ import React, { Component, PropTypes } from 'react';
 class DimensionEntry extends Component {
   constructor (props) {
     super(props);
-    this.state = {
-      isSelected: false
-    };
 
     this.handleClick = this.handleClick.bind(this);
+    this.hidePointerCursor = this.hidePointerCursor.bind(this);
   }
 
+  hidePointerCursor () {
+    const { isInEditMode } = this.props;
+    if(!isInEditMode){
```

Comment:
is this functionality still needed? was kind of hoping we could get rid of these checks once we started using the qlik selection api instead.

Reply:
yes it's still needed
its only job is to show the normal mouse cursor when hover on dimension button in Edit Mode